hub/apps/desktop/modernize/host-custom-control-with-xaml-islands-cpp.md

```diff
@@ -279,12 +279,12 @@ Next, revise the default **App** class in the **MyUWPApp** project to derive fro
             }
         private:
```

Comment:
Can't comment on the exact line since it isn't part of the changes, but on line 174 there's a typo that says `IXamlMetadaraProvider` instead of `IXamlMetadataProvider`